niceday-api/api/openapi.yaml

```diff
@@ -70,6 +109,52 @@ components:
           description: Message to send
           example: Hello World
       description: A Message to Send.
+    SmokingTrackerEntry:
+      type: object
+      properties:
+        id:
+          type: string
+          description: uid of tracker entry
+          example: '620381d1af7a92028c330734'
+        userId:
+          type: integer
+          description: id of user
+          example: 38527
+        startTime: 
+          type: string
+          example: '2022-02-09T08:56:44.349Z'
```

Comment:
Is the timestamp guaranteed to have this datetime format?

Reply:
I guess I mean, will it fail gracefully/safely if the format is ever different?

Reply:
Currently there is no validation at all, see https://github.com/PerfectFit-project/virtual-coach-issues/issues/200 for next step in that direction. So this is just an indication/documentation. I would expect the api have this format for now.

Reply:
I added the `format: date-time` though. I think this will be checked then when we validate the output.